jupyterkernel/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+cmake_minimum_required(VERSION ${CADABRA_CMAKE_VERSION})
+project(Cadabra)
+
+print_header("Configuring Py Cadabra Jupyter Kernel")
+set(
+  PY_CADABRA_JUPYTER_VERSION "1.0.1"
+)
```

Comment:
Unsure how you'd like to have version values controlled, so have just put inplace for now.